microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md

```diff
@@ -48,12 +48,22 @@ Uploads a package to update an existing draft submission of an add-on product.
 
 The following request headers are required:
 
-##### [v2](#tab/v2)
+##### [v1.1](#tab/v1-1)
 
 * `Authorization: ApiKey <api key>`
 * `X-ClientID: <client ID>`
 * `Content-Type: application/zip`
 
+<!-- todo: 
```

Comment:
change <client ID> to $ClientID ?
change <api key> to $ApiKey ?
change order?
per using.md:
Authorization: ApiKey $ApiKey; Content-Type: application/zip; X-ClientID $ClientID
global in article, & check using.md

https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/addons-api-reference?branch=pr-en-us-3269&tabs=v1-1

Reply:
ans: change them to $